deploy/runtime/webapis.conf

```diff
@@ -4,7 +4,7 @@
 android_build=https://sam.nrel.gov/sites/default/files/content/mobile/android/readme.html
 forum=https://sam.nrel.gov/support
 ios_build=https://sam.nrel.gov/sites/default/files/content/mobile/ios/readme.html
```

Comment:
Is this the same as PR#58 into SAM-private?

Reply:
Yes, SAM-private PR#58 is supposed to go with this one.